### ISSUE TITLE
Light gun cursors not visible when hardware rendered

### ISSUE THREAD

**Issue by Sanaki** [libretro/beetle-psx-libretro#643]
## Description

The cursors for light guns aren't visble with vulkan or OpenGL rendering, making upscaling non-viable for light gun games. They do work fine with the software renderer enabled.

### Version/Commit

Beetle PSX HW 0.9.44.1 dfab19a

### Core options

Tons of option changes, but I reproduced the issue with defaults as well.

### Environment information

- Frontend: Retroarch 1.8.5 libretro/RetroArch@21f15d5
- OS: Linux Mint 19.3, kernel 5.3.0-46
- Compiler: GCC 9.3.0 64-bit
- GPU: GTX 1060 6GB, Nvidia driver 440.82

**Comment by dudesh34** [comment]
Yes...i can report also the light gun support it is broken in HW.

**Comment by Tovarichtch** [comment]
Two years later, problem is still present. The only thing that works with light guns are the triggers. No cursor / aiming whatsoever with OpenGL / Vulkan.
